### DISCUSSION TITLE
How to get the HWND of bevy app

### DISCUSSION THREAD

**Discussion by yk0n9** [bevyengine/bevy#8146] [Q&A]
I need to manipulate the actual device using the window handle

**Comment by flmng0** [comment]
Not sure from start to finish how to get it, but if you're using the `WinitPlugin` (which is in `DefaultPlugins`), then you can retrieve the [`RawHandleWrapper`](https://docs.rs/bevy/latest/bevy/window/struct.RawHandleWrapper.html) for a given window.

For a lead, try looking through the `bevy_winit` crate, the windows are created [here](https://github.com/bevyengine/bevy/blob/396c2713a622626c10c5ff81a07a8396134a6579/crates/bevy_winit/src/system.rs#L35).

Check out the `raw_window_handle` crate, which is what is stored in the `RawHandleWrapper`, btw.

Hope this helped, sorry I can't give you a full code example.
